draft/0000-name-main-command-django.rst

```diff
@@ -0,0 +1,184 @@
+========================================
+DEP XXXX: Name the main command `django`
+========================================
+
+:DEP: XXXX
+:Author: Ryan Hiebert
+:Implementation Team: Ryan Hiebert
+:Shepherd: Tom Carrick
+:Status: Draft
+:Type: Feature
+:Created: 2025-01-07
+
+.. contents:: Table of Contents
+   :depth: 3
+   :local:
+
+
+Abstract
+========
+
+Motivated by a desire to remove confusing papercuts in Django
+and to follow common convention in the Python ecosystem,
+this DEP proposes to add a new ``django`` command equivalent to
+the existing ``django-admin`` command,
+and to update the documentation to prefer this new spelling.
+
+Specification
+=============
+
+The ``django`` command will be added as the preferred spelling
+for the existing ``django-admin`` command.
+
+.. code-block:: diff
+
+   --- a/pyproject.toml
+   +++ b/pyproject.toml
+   @@ -45,2 +45,3 @@
+    [project.scripts]
+   +django = "django.core.management:execute_from_command_line"
+    django-admin = "django.core.management:execute_from_command_line"
+
+Official documentation will be updated
+to reference this new ``django`` command
+everywhere that ``django-admin`` is currently referenced.
+The bulk of documentation should be be updated at once
+to ensure that the documentation gives a consistent message.
+
+Because consistency in vocabulary is important
+to help everyone have a clear mental model of the project,
+reasonable effort will be made to update all internal references
```

Comment:
Reasonable effort seems almost like giving up in advance. I don't think it's difficult to say this will happen, rather than committing only to a "reasonable effort"

Reply:
I started a pass at it, and I hit a sharp edge that I wasn't sure what to do with it, and that's what caused me to write this. There are internal documentation references to `djadmin` which refers to the `django-admin` command. I'm a bit unfamiliar with Sphinx and how that reference might be relied on, so I'm worried about changing it, but it also seems like it should be changed for consistency. Still, it feels like that shouldn't be necessary to complete before the initial implementation could be ready for merge.

Reply:
I'm going to remove the section. My fears are the fine details that will have to be worked out with reviewers.